docs/source/changes.rst

```diff
@@ -12,3 +12,9 @@ All previous releases should still be available on pip.
 ------------------
 
 This is hydra-zen's first stable release on PyPI!
+Although we have not yet released version `v1.0.0`, it should be noted that hydra-zen's codebase is thoroughly tested.
+Its test suite makes keen use of the property-based testing library `Hypothesis <https://hypothesis.readthedocs.io/en/latest/>`_. 
+Furthermore, 100% code coverage is enforced on all commits into `main`.
+
+We plan to have an aggressive release schedule for compatibility-preserving patches of bug-fixes and quality-of-life improvements (e.g. improved type annotations).
+hydra-zen will maintain wide a window of compatibility with Hydra versions; we test against pre-releases of Hydra and will maintain compatibility with its future releases.
```

Comment:
hydra-zen will maintain a window of compatibility with Hydra versions; we test against pre-releases of Hydra and will maintain compatibility with its future releases.

Reply:
wait, i'm new to adding the commit in review.  I was just trying to update the last sentence.

Reply:
Use this button to suggest an edit: 
![image](https://user-images.githubusercontent.com/29104956/126233080-2943577a-97d3-4016-b308-e5662c3db0f5.png)